### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by begali** [arkivanov/Decompose#717]
Update kotlin dependency to 2.0.0 and compose to 1.6.10 which is beta version for ios.

**Comment by arkivanov** [comment]
Thanks! Are you blocked by this update somehow? Or is it just nice to have?

**Comment by begali** [comment]
Thanks for quick reply. No. I'm not blocked. Previously, when I changed compose version, I had byte code conflicts with compose cupertino library.  I thought there will be byte code conflicts when migrating to kotlin 2.0.0 and compose 1.6.10. Now, migrated the project. There was no problem.
